### DISCUSSION TITLE
Can I hide a proposal?

### DISCUSSION THREAD

**Discussion by bitsandtea** [snapshot-labs/snapshot#915] [Q&A] [answered]
Hello, I wanted to know if I can hide a proposal. I think there was a field before where we could enter proposal ID's to hide. Does that option exist anymore?

If not, is it possible to make the Core view default on the front page instead of All Proposals?

Thanks

**Comment by bonustrack** [comment, marked as answer]
Hi @bitsandtea this option does not exist anymore what can only archive a proposal if you are admin of the space or author of the proposal. It's also not possible to define the default view we can make a feature request for this.

**Reply by bitsandtea** [reply]
Thanks for the answer @bonustrack . If we wanted to archieve it as an ADMIN, how can we do it? I can't seem to find the option - thanks

**Comment by bonustrack** [comment]
It's here 
![image](https://user-images.githubusercontent.com/16245250/138076172-b1982512-623b-490f-a1f7-4581a7e15795.png)

**Reply by bitsandtea** [reply]
@bonustrack  Beautiful, just another noobie question - is there a way to see who is clearly the admin? Or is that the owner of the ENS?

**Reply by samwhosung** [reply]
@bitsandtea You can go to "About" and see the space admins. Admins can be set in the space settings by the space owner (ENS owner)
